### ISSUE TITLE
Review onrr.gov policy 

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#1812]
To be posted on onrresource 508 page.

**Comment by Maroyafaied** [comment]
@cthomasONRR @mcharg @lpgoldstein please review, I will post this on the wiki when it's final. https://doimspp.sharepoint.com/sites/NRRD/Shared%20Documents/General/ONRR.gov/508/onrr.gov%20accessibility%20policy%20.docx?web=1

**Comment by cthomasONRR** [comment]
@Maroyafaied Looks good!

**Comment by mcharg** [comment]
@Maroyafaied I reviewed and added a comment about ensuring that manual checks are the responsibility of the document owner.

**Comment by lpgoldstein** [comment]
@Maroyafaied I just made a few grammar changes. Looks good!

**Comment by Maroyafaied** [comment]
@mcharg I added this sentence, feel free to make changes to it. 

For any documents requested to be uploaded on onrr.gov, the ODDD team will utilize automated accessibility testing tools to validate accessibility or identify issues. It is the document’s owner responsibility and the law to manually examine and identify issues that cannot be detected by the automated testing tools.

**Comment by mcharg** [comment]
@Maroyafaied Love it!

**Comment by Maroyafaied** [comment]
@mcharg Awesome, did you need me to send this to Jae or just add it to the wiki?

**Comment by mcharg** [comment]
You can just add it to the wiki. I can put a copy of the doc in the folder for Jae.

**Comment by mcharg** [comment]
Uploaded to Teams. https://doimspp.sharepoint.com/:w:/r/sites/NRRD/Shared%20Documents/General/ONRR%20508%20Compliance%20Approach%20Team/onrr.gov%20accessibility%20policy%20.docx?d=w0820bf1de5a340149466d4d20af05686&csf=1&web=1&e=XgOf2o

**Comment by mcharg** [comment]
@Maroyafaied I'll also send it to Dave, so he can copy it for ONRResource.

**Comment by Maroyafaied** [comment]
Added the policy to the wiki. https://github.com/ONRR/onrr.gov-site/wiki/Accessibility-Policy
